### PR TITLE
Add markprompt extension

### PR DESCRIPTION
**Pull request by nickrttn**
## Description

The Markprompt extension enables users of [Markprompt](https://markprompt.com) to prompt their Markprompt projects through Raycast. This allows support teams to ask quick questions of the project content their Markprompt bot has been given as context.

## Screencast

https://github.com/raycast/extensions/assets/2504906/c5f240bd-6640-44d6-b54b-6c49daa82e58

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/markprompt/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "lint": "ray lint",
+    "publish": "npx @raycast/api@latest publish"
+  },
+  "preferences": [
+    {
+      "name": "token",
+      "description": "Get a token in the Markprompt Project settings",
+      "type": "password",
+      "required": true,
+      "title": "Project token"
```
Comment (pernielsentikaer) [inline extensions/markprompt/package.json:52]:
```suggestion
      "title": "Project Token"
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Maybe add a List.EmptyView when entering the extension OR search searchbar placeholder to help people understand how to use it.

- The extension logo doesn't work well in light appearance, you can have an inverted copy and add @dark to the filename
image

- Look at the CI error 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by nickrttn** [comment]
Thanks folks :)

**Pull request merged**